views/pages/single_fixture.js

```diff
@@ -374,18 +376,30 @@ function handleChannel(channel, mode) {
   }
 
   if ('fineChannelAliases' in channel) {
-    const fineChannelsUsedInMode = channel.fineChannelAliases.filter(chKey => mode.channels.includes(chKey));
-
-    if (fineChannelsUsedInMode.length > 0) {
+    const fineChannelPositions = channel.fineChannelAliases.map(
+      fineAlias => mode.channels.findIndex(
+        chKey => chKey === fineAlias ||
+          (chKey in switchingChannels && switchingChannels[chKey].switchedChannels.includes(fineAlias))
+      ) + 1
+    )
+    // the tests make sure that only the first x finenesses can be used in a mode
```

Comment:
This comment is pretty lengthy. Please try to shorten it and move it after the statement (same line) instead of in between.

Something like `leaving out is only allowed at the end -> still continuous array`